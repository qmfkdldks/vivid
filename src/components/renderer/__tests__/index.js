import React from "react";
import { shallow } from "enzyme";
import Renderer from "../index";

describe("Renderer", () => {
  it("should render successfully", () => {
    const component = shallow(<Renderer />);
    expect(component).toMatchSnapshot();
  });
});
