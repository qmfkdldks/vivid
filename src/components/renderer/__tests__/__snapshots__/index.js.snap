// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderer should render successfully 1`] = `
<Slate
  editor={
    Object {
      "addMark": [Function],
      "apply": [Function],
      "children": Array [],
      "deleteBackward": [Function],
      "deleteForward": [Function],
      "deleteFragment": [Function],
      "getFragment": [Function],
      "insertBreak": [Function],
      "insertData": [Function],
      "insertFragment": [Function],
      "insertNode": [Function],
      "insertText": [Function],
      "isInline": [Function],
      "isVoid": [Function],
      "marks": null,
      "normalizeNode": [Function],
      "onChange": [Function],
      "operations": Array [],
      "removeMark": [Function],
      "selection": null,
      "setFragmentData": [Function],
    }
  }
>
  <Editable
    readOnly={true}
    renderElement={[Function]}
    renderLeaf={[Function]}
  />
</Slate>
`;
